Detections/ASimDNS/imDNS_Miners.yaml

```diff
@@ -18,6 +18,8 @@ relevantTechniques:
 tags:
   - Id: 0d76e9cf-788d-4a69-ac7d-f234826b5bed
   - version: 1.0.0
+  - Schema: ASIMDns
```

Comment:
Current schema version for ASIMDns is 0.1.1

Reply:
Thanks. also fixed on the MS parser